indico/modules/designer/templates/template.html

```diff
@@ -271,9 +318,14 @@
             var template = {
                 title: {{ template.title | tojson }},
                 data: {{ template.data | tojson }},
-                background: {{ (template.background_image.download_url if template.background_image else none) | tojson }}
+                background_url: {{ (template.background_image.download_url if template.background_image else none) | tojson }}
+            };
+            var backsideTemplate = {
+                title: {{ (template.backside_template.title if template.backside_template else none) | tojson }},
+                data: {{ (template.backside_template.data if template.backside_template else none) | tojson }},
+                background_url: {{ (template.backside_template.background_image.download_url if template.backside_template and template.backside_template.background_image else none) | tojson }}
```

Comment:
this line is awfully long.. might be nicer to build the whole `backside_template` dict in python and then just jsonify it here:

    var backsideTemplate = {{ backside_template | tojson }};